### ISSUE TITLE
All Rights Reserved

### ISSUE THREAD

**Issue by Danie10** [geekitude/piwigo-plugin-simplecopyright#19]
I've only noticed this on a recent upload of photos a few days ago, that now the copyright notice shows All Rights Reserved on the EXIF data for the photo. I use a Creative Commons license, which is set as my default. So no idea why/where this comes from. Older photos, I see shoe N/A for copyright on the photo EXIF data shown.

I tried installing a plugin called Copyrights, which lets me set/specify a copyright, but now it shows a duplicate copyright notice (correct one) but the All Rights Reserved one still remains. Tried writing metadata to those photos, but no change. My default copyright footer notice is still correct.

I'm not sure if this is something that was affected by the Simple Copyright plugin, but any advice would be appreciated.

**Comment by geekitude** [comment]
Hi

Can you confirm Simple Copyright installed version and post a few
screenshots here?
- your Simple Copyright settings
- what piwigo shows as Copyright on an image page?

Simon

**Comment by Danie10** [comment]
OK thanks it is version 13a of the plugin. I checked original photo that was uploaded, and its EXIF data has no copyright text, neither under IPTC. The screenshot shows the now duplicate notices, but the top one is from that extra plugin I just installed to try set a notice. Second one is the original copyright notice we used to see - but it is stating All Rights Reserved. So that must have been set during an upload.

![Screenshot_20221102_122927](https://user-images.githubusercontent.com/1153726/199467422-0eb6b27e-ff80-4f08-aa87-512633c10ea2.jpg)

![Screenshot_20221102_123018](https://user-images.githubusercontent.com/1153726/199467434-c9fc0842-cda9-472c-b81c-18e92b4637e3.jpg)

**Comment by Danie10** [comment]
I think issue is with the plugin, as I disabled that option "Display image page Copyright section." and then that notice disappears with the All Rights Reserved. So it could be that it is defaulting to All Rights Reserved instead of the default I had given it.

Footer is displaying correct default though.

**Comment by geekitude** [comment]
Yes, it's most likely a problem with Simple Copyright, sorry for that :)

I'll investigate as soon as possible.

**Comment by Danie10** [comment]
OK thanks. As a work around the Copyrights plugin is doing the job for now. Then when your update comes out and I can just remove the Copyrights plugin.
